dune.module

```diff
@@ -16,5 +16,5 @@
 Module: dune-gdt
 Version: 18.11-dev
 Maintainer: felix.schindler@wwu.de
-Depends: dune-localfunctions dune-xt-grid (>= 18.11) dune-xt-la (>= 18.11) dune-xt-functions (>= 18.11)
+Depends: dune-localfunctions dune-xt (>= 18.11)
 Suggests: dune-fem (>= 1.3)
```

Comment:
Just a side note: do we even have anything depending on dune-fem left in dune-gdt? I could not find a single include. If not, we should also remove the suggestion in the `dune.module`.

Reply:
Probably not, I will remove the suggestion.